ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-redesigned.component.html

```diff
@@ -0,0 +1,208 @@
+<main class="main main_activity"
+      [ngClass]="{
+          'main_sticky': isLoaded && model && model.subtitle,
+          'with-vertical-progress-indicator': isVerticalProgress
+      }"
+>
+    <ng-container *ngIf="isLoaded && model">
+        <section class="section">
+            <ddp-subject-panel></ddp-subject-panel>
+            <ddp-admin-action-panel
+                [activityReadonly]="isReadonly()"
+                (requestActivityEdit)="updateIsAdminEditing($event)">
+            </ddp-admin-action-panel>
+        </section>
+        <section *ngIf="model.subtitle" class="section sticky-section"
+                 [ngClass]="{'sticky-section_shadow': isScrolled}">
+            <div class="content content_tight">
+                <div class="sticky-block" [innerHTML]="model.subtitle"></div>
+            </div>
+        </section>
+        <section *ngIf="model.title" class="section header-section">
+            <div class="content content_tight">
+                <h1 class="activity-header" [innerHTML]="model.title"></h1>
+            </div>
+        </section>
+    </ng-container>
+
+    <!-- article content -->
+    <section *ngIf="isLoaded; else spinner" class="section article-content">
+        <ng-container *ngIf="isVerticalProgress && isStepped && showStepper; then verticalProgressIndicator;">
+        </ng-container>
```

Comment:
Using the vertical progress indicator there (as an external block on the left of a main article content)